donkeycar/parts/tub_v2.py

```diff
@@ -75,9 +75,11 @@ def delete_records(self, record_indexes):
         self.manifest.delete_records(record_indexes)
 
     def delete_last_n_records(self, n):
-        last_index = self.manifest.current_index
-        first_index = max(last_index - n, 0)
-        self.manifest.delete_records(range(first_index, last_index))
+        # build ordered list of non-deleted indexes
+        all_alive_indexes = sorted(set(range(self.manifest.current_index))
```

Comment:
nit: call this `live_indexes`. It reads better.

Reply:
Yeah, will probably do this as part of next change, so I can merge this now.